man/systemd-repart.xml

```diff
@@ -364,6 +364,16 @@
         <xi:include href="version-info.xml" xpointer="v252"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--signing-engine=</option></term>
+
+        <listitem><para>Takes a signing engine identifier. Currently only OpenSSL engines and providers are
+        supported, such as <literal>pkcs11</literal>. When specified, the value of
+        <option>--private-key=</option> is passed to the signing engine verbatim.</para>
+
+        <xi:include href="version-info.xml" xpointer="v255"/></listitem>
+      </varlistentry>
```

Comment:
can you add an example to the end that shows how to use this for example with a yubikey as backing